manifest.json

```diff
@@ -38,7 +38,8 @@
 	],
 	"externally_connectable": {
 		"matches": [
-			"https://ethsbell.app/*"
+			"https://ethsbell.app/*",
+			"https://dayschool.ethsbell.app/*"
```

Comment:
Shouldn't we add day school to the permissions above this as well?

Reply:
No, and I actually don't think we even need `https://ethsbell.app/*` there in the first place.
When a host is specified in `permissions` it's considered a "host permission" which lets you do things like modify the page content, intercept network requests, and [make requests to the domain that ignore cross-origin restrictions](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/host_permissions#:~:text=XMLHttpRequest%20and%20fetch%20access%20to%20those%20origins%20without%20cross%2Dorigin%20restrictions).
We don't actually do any of that, and our servers are set to allow requests from any domain.

As for why we need to add the day school domain to `externally_connectable`, it's to let the sites push schedule configs to the extension.

In my testing, there was never a problem with anything, so I'm pretty confident it isn't necessary.